test/pretty-printer/ppTest.ml

```diff
@@ -11,14 +10,17 @@ let generator =
            let lowercase_id =
              string_size ~gen:(char_range 'a' 'z') (return 1)
            in
-           let regular_id id = Id.R.mk id in
-           let modal_id id = Id.M.mk (id ^ "'") in
+           let regular_id id = Mtt.Id.R.mk id in
+           let modal_id id = Mtt.Id.M.mk (id ^ "'") in
```

Comment:
Is it possible to not prefix this with `Mtt`?

Reply:
Yes, it's my bad with merge. Fixed :)